sa/_db/migrations/20150826133823_SCTReceipts.sql

```diff
@@ -0,0 +1,22 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE `sctReceipts` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `sctVersion` tinyint(3) NOT NULL,
+  `logID` varchar(255) NOT NULL,
+  `timestamp` bigint(20) NOT NULL,
+  `extensions` mediumblob,
+  `signature` mediumblob,
+  `certificateSerial` varchar(255) NOT NULL,
+  `LockCol` bigint(20) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `log_serial` (`logID`,`certificateSerial`)
```

Comment:
Actually, an index with multiple keys can be queried by a single column, so long as that column is first in the index. So to support a `SELECT * where certificateSerial = "xyz"` query, you want an index like:

```
UNIQUE KEY `certificateSerial_logID` (`certificateSerial`, `logID`)
```

It's probably worth setting a convention that indexes are always named after the full name of the columns they index, concatenated with `_`.